packages/primitives/src/Function.ts

```
@@ -0,0 +1,99 @@
+import { Selector, type Selector as SelectorType } from "./Selector.js";
+import { B256, type B256 as B256Type } from "./B256.js";
+
+/**
+ * Ethereum ABI function pointer: address (20 bytes) + selector (4 bytes)
+ */
+export type Function = FixedBytes<24>;
+
+/**
+ * Schema for validating Function values from Uint8Array
+ */
+export const Function = fixedBytesFromBytes(24);
```

Biome (1.9.4)

[error] Do not shadow the global "Function" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)